talent/application-process.md

```diff
@@ -52,22 +65,24 @@ The interviews consist of the following:
 
 - Design interview (90min): to see how the developer would design a solution to a given technical problem working with several of our engineers.
 
-* Q&A (60min): to get to know the developer better. Two interviewers should participate; one from the team that the developer is being interviewed for, and one from any other engineering or non-engineering team (eg. Product, Talent). Duration 60'.
+* Q&A (60min): to get to know the developer better. Two interviewers should participate; one from the team that the developer is being interviewed for, and one from any other engineering or non-engineering team (eg. Product, Talent).
 
 - Logical thinking (60min): to see how the developer solves an abstract problem in a logical way.
 
-- In the case of rejection, the developer should be notified in person on the day by the lead of the corresponding team. This should be a constructive discussion giving the developer as much insight as possible. 
+- In the case of rejection, the candidate will be notified in the following days after the interview. This will be a constructive discussion giving the candidate as much insight as possible. 
 
 ### Offer
 
-- The offer is sent by our CTO through Lever.
+- The offer is sent by our CTO or Talent Acquisition Manager once a decision to hire has been made by the team
+
+- The candidate has no concrete time deadline to accept/reject the offer, and we are happy to have another call to discuss any questions or doubts the candidate may have. In cases where we have a limited number of open positions for a role, and a larger number of people we'd like to make an offer to, a time limit is given.
```

Comment:
Should we also mention that we may also impose a deadline if we are legally required to do so?

Reply:
Yes this point needs discussing

Reply:
Perhaps we can document this without making a decision on policy by writing something like:

"Except in cases where the law requires otherwise, the candidate has no fixed time deadline …"

or words to that effect? This does not commit us to a particular strategy but does clarify the possibility.